rotkehlchen/api/rest.py

```diff
@@ -2557,6 +2557,14 @@ def purge_exchange_data(self, location: Optional[Location]) -> Response:
 
         return api_response(OK_RESULT, status_code=HTTPStatus.OK)
 
+    @require_loggedin_user()
+    def get_connected_locations(self) -> Response:
+        locations = self.rotkehlchen.data.db.get_known_locations()
```

Comment:
Choose a name please. You use both known and connected locations. I am much more inclined to go with connected locations here.